bin/genome-env

```diff
@@ -541,6 +585,8 @@ default_value TESTDBSERVER_DB_USER "genome"
 default_value TESTDBSERVER_DB_PASS "mypassword"
 default_value TESTDBSERVER_URL "https://apipe-test-db.gsc.wustl.edu"
 default_value GE_ENV_NAME "genome-env"
+default_value TESTDBSERVER_DB_HOST "apipe-test-db.gsc.wustl.edu"
+default_value TESTDBSERVER_DB_PORT "5432"
```

Comment:
I agree - those were from before when we were trying to just pass in the db name with -i; I should have deleted them!